### ISSUE TITLE
Mumbling Voice

### ISSUE THREAD

**Issue by Blandrust** [Edresson/YourTTS#40]
Hey there,

I'm trying to fine-tune the TTS model for the German language, but I'm fairly new to this field. I've tried various approaches and datasets, such as the German part of the M-AI Labs dataset, and the Mozilla Common Voice dataset. I've also adjusted the Coqui recipe you provided to fit with the new datasets.

I've tried training the model for 100k steps and even up to 500k steps, but I can't seem to get the model to learn German. It's always kind of mumbling and not learning anything at all, even though the loss is still decreasing after 100k steps and onwards.

Has anyone experienced this kind of issue before? Do I need to train the model for a longer period, or am I missing something crucial in my approach?

Thanks in advance.

**Comment by harshvardhan-truefan** [comment]
Hi, 
I'm facing the same issue as well. I tried to fine-tune the yourtts model on my dataset, but the test audios that I'm getting in the end is a person mumbling, and is not able to form sentences. Any idea about how to solve this issue ? 

Thanks and Regards,
Harsh

**Comment by thivux** [comment]
did anyone find the solution?
